database/seed.js

```diff
@@ -33,7 +33,7 @@ function generateData() {
   }
   return games;
 }
-var test = generateData();
+var seededData = generateData();
```

Comment:
I like the name change to a variable that makes more sense as someone new to your codebase. Are you using a linter yet? I am seeing a lot of Let and Const which is good. Why is this line using Var?